julia-package/brainflow/test/transforms.jl

```diff
@@ -1,12 +1,12 @@
-import brainflow
+using brainflow
```

Comment:
Just curios whats the difference between using and import

Reply:
https://docs.julialang.org/en/v1/manual/modules/#Summary-of-module-usage

`import` only brings the package into the scope. `using` also brings the explicitly exported functions and types into scope.